stripe/http_client.py

```diff
@@ -187,10 +207,31 @@ def _handle_request_error(self, e, url):
         msg = textwrap.fill(msg) + "\n\n(Network error: " + str(e) + ")"
         raise error.APIConnectionError(msg)
 
-
 class PycurlClient(HTTPClient):
     name = 'pycurl'
 
+    def __init__(self, verify_ssl_certs=True, proxies= None):
+        super( PycurlClient, self ).__init__( verify_ssl_certs= verify_ssl_certs, proxies= proxies )
```

Comment:
Should be `super(PycurlClient, self).__init__(verify_ssl_certs=verify_ssl_certs, proxies=proxies)`